src/App.js

```diff
@@ -1,13 +1,49 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+
+  constructor() {
+    super();
+
+    this.state = {
+      todoList: [
+        {
+          task: 'Organize Garage',
+          id: 1528817077286,
+          completed: false
+        },
+        {
+          task: 'Bake Cookies',
+          id: 1528817084358,
+          completed: false
+        },
+      ]
+    };
+  };
+
+  clickHandle = (event) => {
```

Comment:
`handleClick` makes more sense to me as a method name. A lot of developers name their functions starting with a verb that describes what the function does